sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -192,7 +192,7 @@ statement
         LIKE pattern=STRING partitionSpec?                             #showTable
     | SHOW TBLPROPERTIES table=tableIdentifier
         ('(' key=tablePropertyKey ')')?                                #showTblProperties
-    | SHOW COLUMNS (FROM | IN) tableIdentifier
+    | SHOW COLUMNS (FROM | IN) multipartIdentifier
         ((FROM | IN) db=errorCapturingIdentifier)?                     #showColumns
```

Comment:
https://mariadb.com/kb/en/library/show-columns/

It seems like when `(FROM | IN) db` is specific, the table name must be a single part. Shall we enforce it as well?

Reply:
BTW, in v2 command, we use `namespace` to replace `database`, as `namespace` is a more general concept. The name of `namespace` is also multi-part, so we should use `multipartIdentifier`.

Reply:
Thank you for the explanation, now I have more clear some concepts. 
I agree with the proposal of maintaining the table name as a single part when namespace is defined.